common/src/main/java/com/wynntils/models/lootruns/LootrunRenderer.java

```diff
@@ -191,6 +202,72 @@ private static void renderNonTexturedLootrunPoints(
         }
     }
 
+    private static void renderTexturedLootrunPoints(
+            PoseStack poseStack, List<ColoredPath> locations, Level level, RenderType renderType) {
+
+        Camera camera = McUtils.mc().gameRenderer.getMainCamera();
+        poseStack.pushPose();
+        poseStack.translate(camera.getPosition().x, camera.getPosition().y, camera.getPosition().z);
+
+        for (ColoredPath locationsInRoute : locations) {
+            VertexConsumer consumer = BUFFER_SOURCE.getBuffer(renderType);
+            List<Pair<ColoredPoint, ColoredPoint>> toRender = new ArrayList<>();
+            boolean sourceBatchEnded = false;
+            BlockPos lastBlockPos = null;
+
+            boolean pauseDraw = false;
+
+            for (int i = 0; i < locationsInRoute.points().size() - 1; i += 10) {
```

Comment:
Why size - 1?

Reply:
second point have to be passed in order to get the direction